content/docs/guides/gke/cloud-filestore.md

```diff
@@ -0,0 +1,89 @@
++++
+title = "Using Cloud Filestore"
+description = "Using Cloud Filestore with Kubeflow"
+weight = 3
+toc = true
+bref = "This guide describes how to set up and use Cloud Filestore with Kubeflow on Google Cloud Platform (GCP)."
+
+[menu.docs]
```

Comment:
good to know! I didn't know menu parent could be configured here in frontmatter.